Latest nightly: SWC regression: Parcel require uses shorthand property names, failing in ie11.

Sorry for bad issue report, had a hard time creating a repro repo because I don't understand what triggers the wrapping of some code in my code.

The issue is on parcel `2.0.0-nightly.686+14a039f0` when building for target `ie11`.

The output code contains this:
```
if (parcelRequire == null) {
  parcelRequire = function(id) {
    if (id in $parcel$modules) {
      return $parcel$modules[id].exports;
    }
    if (id in $parcel$inits) {
      let init = $parcel$inits[id];
      delete $parcel$inits[id];
      let module = {id, exports: {}};
      $parcel$modules[id] = module;
      init.call(module.exports, module, module.exports);
      return module.exports;
    }
    var err = new Error("Cannot find module '" + id + "'");
    err.code = 'MODULE_NOT_FOUND';
    throw err;
  };

  parcelRequire.register = function register(id, init) {
    $parcel$inits[id] = init;
  };

  $parcel$global["parcelRequirea9f6"] = parcelRequire;
}
```

And this throws an error in ie11:
```
      let module = {id, exports: {}};
```
because it has to be `id: id` for ie11 to understand it.
      
@mischnic is this enough for you to understand and fix? 👀 The issue is similar to https://github.com/parcel-bundler/parcel/issues/5555

--- a/packages/packagers/js/src/helpers.js
+++ b/packages/packagers/js/src/helpers.js
@@ -13,7 +13,7 @@ if (parcelRequire == null) {
     if (id in $parcel$inits) {
       let init = $parcel$inits[id];
       delete $parcel$inits[id];
-      let module = {id, exports: {}};
+      let module = {id: id, exports: {}};
       $parcel$modules[id] = module;
       init.call(module.exports, module, module.exports);
       return module.exports;

